setup.cfg

```diff
@@ -0,0 +1,192 @@
+[bdist_wheel]
+universal = 1
+
+[metadata]
+name = farm-haystack
+version = file: VERSION.txt
+url = https://github.com/deepset-ai/haystack
+project_urls =
+    Docs: RTD = https://haystack.deepset.ai/overview/intro
+    CI: GitHub = https://github.com/deepset-ai/haystack/actions
+    GitHub: issues = https://github.com/deepset-ai/haystack/issues
+    GitHub: repo = https://github.com/deepset-ai/haystack
+description = Neural Question Answering & Semantic Search at Scale. Use modern transformer based models like BERT to find answers in large document collections
+long_description = file: README.md
+long_description_content_type = text/markdown
+keywords=
+    QA 
+    Question-Answering 
+    Reader 
+    Retriever 
+    semantic-search 
+    search 
+    BERT 
+    roberta 
+    albert 
+    squad 
+    mrc 
+    transfer-learning 
+    language-model 
+    transformer
+author = Malte Pietsch, Timo Moeller, Branden Chan, Tanay Soni
```

Comment:
Please add Julian, Thomas, Bogdan, Michel and yourself as authors :)

Reply:
I was thinking, maybe I can put "deepset.ai" as author? Or the list will grow long in the next year :grin: